src/DIRAC/Resources/Computing/ARCComputingElement.py

```diff
@@ -79,9 +95,80 @@ def __init__(self, ceUniqueID):
 
     #############################################################################
 
+    def _isThereRestInterface(self):
+        # The name is the documentation. Importantly, if all is well the following variables are set
+        # - and a few others on which they (especially self.s) depend.
+        # self.rest    : Static variable to say if the REST interface is available and ready
+        # self.base_url: The basic URL for the REST interface
+        # self.s       : The python "requests" module session
+        # self.headers : The format of information we prefer : json of course
+        #
+        self.log.debug("Testing if the REST interface is available for %s" % self.ceHost)
+
+        # Is this needed? Test if the CE is supposed to be an ARC CE.
+        result = getCESiteMapping(self.ceHost)
+        if not result["OK"] or not result["Value"]:
+            self.log.error("Unknown CE ...")
+            return
+        self.site = result["Value"][self.ceHost]
+        grid = self.site.split(".")[0]
+        ceString = "Resources/Sites/%s/%s/CEs/%s/CEType" % (grid, self.site, self.ceHost)
+        print(ceString)
+        result = gConfig.getValue(ceString, defaultValue="")
+        if result != "ARC":
+            self.log.info("This ce (%s) is not an ARC CE. It is : %s. Returning." % (self.ceHost, result))
+            self.rest = False
+            return
+
+        # Is the ARC endpoint published / available. It is also a check on the CE health.
+        self.log.debug("First check if the CE %s publishes its REST endpoint" % self.ceHost)
+        command = """ ldapsearch -x -LLL -h %s:2135 -b 'o=glue' \
+                  GLUE2EndpointInterfaceName=org.nordugrid.arcrest GLUE2EndpointURL \
+                  | grep GLUE2EndpointURL | awk -F ": " '{print $2}' """
+        status = shellCall(1, command % self.ceHost)
```

Comment:
Which URL is queried here?

Also: we have the BDII2CSAgent: https://github.com/DIRACGrid/DIRAC/blob/rel-v7r2/src/DIRAC/ConfigurationSystem/Agent/Bdii2CSAgent.py so I am wondering if we should not instead just look in CS for this option. CC @andresailer